React/Views/RCTPicker.m

```diff
@@ -12,6 +12,9 @@
 #import "RCTUtils.h"
 
 @interface RCTPicker() <UIPickerViewDataSource, UIPickerViewDelegate>
+
+ #define UIColorFromRGB(rgbValue) [UIColor colorWithRed:((float)((rgbValue & 0xFF0000) >> 16))/255.0 green:((float)((rgbValue & 0xFF00) >> 8))/255.0 blue:((float)(rgbValue & 0xFF))/255.0 alpha:1.0]
```

Comment:
There's an example in MapView, where we use it for the pin image.

I wonder if it mightn't be simpler just to include the item styles in the main stylesheet though. If you just put font, textColor, etc. on the main component, it's unambigous that they refer to the labels in the items, and then you can export the properties individually, like we do for RCTText, which handles the color conversion automatically.